Aquifer.Common.UnitTests/BibleWordIdentifierTests.cs

```diff
@@ -0,0 +1,346 @@
+using Aquifer.Common.Utilities;
+using Aquifer.Data.Enums;
+
+namespace Aquifer.Common.UnitTests;
+
+public sealed class BibleWordIdentiferTests
```

Comment:
The new `BibleWordIdentifier` class is needed so that I can easily create word identifiers and also parse word identifiers.  There are a decent number of methods on the new class with some complicated math so I decided to write an exhaustive suite of unit tests.   This didn't take too long because many tests are similar but just hit a different constructor overload.

Reply:
Typo in Identifier

Reply:
Thanks.  I used to get IDE warnings for the class name not matching the file name but maybe that was Resharper, not VS.